packages/react-components/stories/in-tests-runner.ts

```diff
@@ -1,2 +1,4 @@
-const inTestsRunner = () => process.env.NODE_ENV === 'PERCY' || process.env.NODE_ENV === 'cypress'
+const inTestsRunner = () => {
+    return process.env.NODE_ENV === 'PERCY' || !!(window as any).Cypress
```

Comment:
@pshoniuk this is a nice way to test for Cypress, there used to be a better way for Percy with an `in-percy` package, but it stopped working and I needed to add the `NODE_ENV` variable

If you know a better way to test for Percy though let's switch